files/config.yml

```diff
@@ -0,0 +1,7 @@
+probes:
+  - name: mqtt broker NONSSL
```

Comment:
I feel like these settings should be configurable (especially topic and client prefix), we have quite a lot of topics as well so it would be good to see if we can subscribe to all.